File: python/Command/SaveCommand.py
from ICommand import ICommand
from FileHandler import FileHandler

class SaveCommand(ICommand):    
    def __init__(self, fileHandler: FileHandler) -> None:
        print("COMMAND: Creating command\n")
        self.fileHandler = fileHandler

    def Execute(self) -> None:
        print("COMMAND: Executing command from invoker, delegating to logic\n")
        self.fileHandler.SaveFile()

File: C#/Command/SaveCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Command
{
    class SaveCommand : ICommand
    {
        private FileHandler fileHandler;        

        public SaveCommand(FileHandler fileHadnler)
        {
            Console.WriteLine("COMMAND: Creating command");
            this.fileHandler = fileHadnler;            
        }

        public void Execute()
        {
            Console.WriteLine("COMMAND: Executing command from invoker, delgating to logic");
            fileHandler.SaveFile();
        }
    }
}
